src/bundle/Controller/Content/ContentTreeController.php

```diff
@@ -72,10 +72,11 @@ public function __construct(
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
      */
     public function loadChildrenAction(
-        Request $request,
         int $parentLocationId,
         int $limit,
-        int $offset
+        int $offset,
+        Query\Criterion $filter,
+        Request $request
```

Comment:
Why you changed position of $request parameter ?

Reply:
Looks like some mistake. Fixed: https://github.com/ibexa/admin-ui/pull/1290/commits/1bdbdcfe64298da1d0c7ffcf2daf4de3ecedcbb5